models/generalinfo.js

```diff
@@ -22,7 +22,19 @@ const GeneralinfoSchema = new Schema({
 		type: String,
 		enum: require("../public/dist/js/categories").categories,
 		required: "General information requires a category"
+	},
+	school: {
+		type: Schema.Types.ObjectId,
+		required: "General information requires a school or to be null"
 	}
 });
 
+GeneralinfoSchema.pre("validate", function (next) {
+	const generalinfo = this;
+	if (generalinfo.school == null) {
```

Comment:
Prefer triple `===` over `==`

Reply:
I found the == to be a bit more useful in this case because the request sometimes has an empty string instead of a null or undefined. The == in this case takes care of both those scenarios. Again though am not an expert maybe I should change it to `if (generalinfo.school === "") `. Whats your opinion?

Reply:
That indeed seems like a better way, ill do that.